### PR TITLE
update Jenkins plugin page

### PR DESCRIPTION
**Pull request by briandesousa**
Updated and expanded instructions on the Jenkins plugin page.

### CODE REVIEW

**Comment by dtuite** [comment]
Deploy preview here: https://deploy-preview-249--roadie.netlify.app/backstage/plugins/jenkins/

File: content/backstage/plugins/jenkins.md [hunk 1]
```diff
@@ -12,54 +12,85 @@ seo:
...
+            <EntityJenkinsContent />
+          </EntityLayout.Route>
           ...
-        }
-      }
+        </EntityLayout>
+      );
 
-  - intro: Add proxy configuration
+  - intro: Add proxy configuration to your Backstage app's config. A Jenkins API token is required.
```
Comment (dtuite) [inline content/backstage/plugins/jenkins.md:42]: It's worth mentioning that there are instructions which explain how to get the token further down the page. You can even use markdown to link to the relevant section.

File: content/backstage/plugins/jenkins.md [hunk 2]
```diff
@@ -12,54 +12,85 @@ seo:
     Build, test, and deploy on Jenkins CI/CD platform.
 
 logoImage: '../../assets/logos/jenkins/logo-jenkins.png'
-coverImage: '../../assets/jenkins-plugin.png'
+coverImage: '../../assets/backstage/plugins/jenkins/jenkins-plugin.png'
```
Comment (dtuite) [inline content/backstage/plugins/jenkins.md:15]: Would you mind switching out the screenshots for light mode ones please? roadie.io only supports light mode and dark mode screenshots look out of place on the white background. 

You can switch your Backstage instance to light mode in the settings.

File: content/backstage/plugins/jenkins.md [hunk 3]
```diff
@@ -12,54 +12,85 @@ seo:
...
     code: |
+      // catalog-info.yaml
       metadata:
         annotations:
-          jenkins.io/github-folder: [folder-name/job-name]
+          jenkins.io/github-folder: '[github-organization-project-name/job-name]
+
 ---
+
+### Jenkins plugin features
```
Comment (dtuite) [inline content/backstage/plugins/jenkins.md:63]: This is all great and useful info for anyone looking to try this plugin.

**Review by dtuite — CHANGES_REQUESTED** [review]

File: content/backstage/plugins/jenkins.md (hunk 2 above)
Comment (briandesousa) [reply content/backstage/plugins/jenkins.md:15]: Ah good point. Will keep this in mind for the mind for future work.

**Review by dtuite — APPROVED** [review]

**Pull request merged**